src/helpers/pivot/pivot_time_adapter.ts

```diff
@@ -77,6 +77,36 @@ const dayOfMonthAdapter: PivotTimeAdapterNotNull<number> = {
   },
 };
 
+/**
+ * normalizes day of week number
+ */
+const dayOfWeekAdapter: PivotTimeAdapterNotNull<number> = {
+  normalizeFunctionValue(value) {
+    const day = toNumber(value, DEFAULT_LOCALE);
+    if (day < 1 || day > 7) {
+      throw new EvaluationError(
+        _t("%s is not a valid day of week (it should be a number between 1 and 7)", day)
+      );
+    }
+    return day;
+  },
+  toValueAndFormat(normalizedValue, locale) {
+    /**
+     * Normalized value is the day of week (1 based index) depending on the
+     * locale week start day. We need to convert it to a 0 based index with 0
+     * being Sunday.
```

Comment:
I'm not sure to understand this.
So normalized value is localized, but the value in the spreadsheet is always 0=Sunday ?